packages/sanity/src/core/releases/tool/Migrate.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-console */
+import {Flex, Stack, Text} from '@sanity/ui'
+import {useState} from 'react'
+
+import {Button} from '../../../ui-components'
+import {useAddonDataset} from '../../studio'
+
+/**
+ * temporary migration script and buttons to trigger it, to add slug to bundles and to remove the bundles names once merged
+ */
+export function MigrateBundlesSlug() {
```

Comment:
Love that you've included a migration! Btw, [migrations using the CLI](https://www.sanity.io/docs/cli-migration) are also a pretty nice way to handle this kind of thing (if you don't already know about them).

Reply:
Yeah, they are great, I have used them before, but not sure how to use them inside the monorepo 😓 that's why I went for this. Also, it gives us the option to trigger it immediately once everyone has updated their branch

Reply:
No problem! I think you can create and execute them inside `dev/test-studio` and they should just work 🙂.